storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -272,8 +272,9 @@ Options
 .. option:: --log-stream
```

Comment:
log-stream option is also a ghost from the past. Perl innobackupex used to run it to mix in redo logs when it was streaming in tar format.